create.go

```diff
@@ -284,3 +292,58 @@ func createPIDFile(pidFilePath string, pid int) error {
 
 	return nil
 }
+
+// copyIfNeeded copies the cpuset.cpus and cpuset.mems from the parent
+// directory to the current directory if the file's contents are 0
+func copyIfNeeded(current, parent string) error {
+	var (
+		err                      error
+		currentCpus, currentMems []byte
+		parentCpus, parentMems   []byte
+	)
```

Comment:
No need to declare those variables here.
Please do the following function call like this:
```go
currentCpus, currentMems, err := getSubsystemSettings(current)
if err != nil {
        return err
}

parentCpus, parentMems, err := getSubsystemSettings(parent)
if err != nil {
        return err
}
```